policies/api/http/metrics.go

```diff
@@ -410,6 +410,28 @@ func (m metricsMiddleware) DeleteSinkFromAllDatasetsInternal(ctx context.Context
 	return m.svc.DeleteSinkFromAllDatasetsInternal(ctx, sinkID, ownerID)
 }
 
+func (m metricsMiddleware) DeleteAgentGroupFromAllDatasets(ctx context.Context, aGroupID string, token string) error {
+	ownerID, err := m.identify(token)
+	if err != nil {
+		return err
+	}
+
+	defer func(begin time.Time) {
+		labels := []string{
+			"method", "deleteAGroupFromAllDatasetsInternal",
```

Comment:
same note on AGroup here. this one should probably match the actual method name anyway